src/modes/overview/vegaSpec.js

```diff
@@ -178,11 +175,12 @@ export function createSpec(sensor, selections, dateRange, valuePatch) {
             type: 'single',
             empty: 'none',
             on: 'mouseover',
-            nearest: false,
+            nearest: true,
+            encodings: ['x'],
             clear: 'mouseout',
           },
         },
-        // use vertical rule for selection, since nearest is a real performance bummer
+        // use vertical rule for selection, since nearest point is a real performance bummer
         mark: {
```

Comment:
do we still need this layer, since nearest is enabled or can we merge the selection back to the point layer below?

Reply:
We do still need the 'highlight' selection, and merging it with the following 'point' mark layer doesn't work unless you are positioned right where the point would be, since the selection is based on the mark itself, so the invisible vertical rule is what is being selected currently.

It also doesn't work to move the selection down to the following 'rule' mark, though I am not sure why.  Maybe order of the layers matters?